test.js

```diff
@@ -1,23 +1,62 @@
-require('dotenv').load()
-const test = require('ava')
-const Deskbookers = require('./dist/index')
+import dotenv from 'dotenv'
```

Comment:
Like I mentioned, this branch isn't finished yet. Including the unit tests. You told me to just commit it so you can finish it up.... So that is what I did. So feel free to start this file from scratch... I just did it to quickly test some code.